.circleci/config.yml

```
@@ -136,21 +95,21 @@ jobs:
             git fetch
             .circleci/is-version-number-acceptable.sh
 
-  deploy_python2:
+  deploy:
     docker:
-      - image: python:2.7.15
+      - image: python:3.7
     environment:
-      PYPI_USERNAME: openfisca-bot
+      PYPI_USERNAME: openfisca-bot  # Edit this value to replace it by your Pypi username
```

@Morendil As-tu un exemple de cas dans lequel nous aurions besoin de déployer via CircleCI avec notre identité propre ? 🤔

---

Tu as raison, je supprime le commentaire. (C'était là d'origine dans la section Python 3 mais pas dans la section Python 2 et c'est pour cela que Github l'affiche comme si c'était un ajout à cette ligne.)

```suggestion
      PYPI_USERNAME: openfisca-bot
```